Capstone Project/ViewControllers/Workouts/CurrentWorkoutViewController.m

```diff
@@ -33,12 +31,17 @@ - (void)viewDidLoad {
     self.tableView.delegate = self;
     self.exerciseTextField.delegate = self;
     
-    self.arrayOfExercises = [[NSMutableArray alloc] init];
+    self.autocompleteTableView.delegate = self;
```

Comment:
This is suggested text. Autocomplete shows up on the same text field (as it is in some chat apps). Suggested text offers multiple options as drop down whereas automcomplete offers only one option. Please rename this to suggestedTextTableView